ocaml-lsp-server/src/state.mli

```diff
@@ -32,6 +32,7 @@ val create :
   -> diagnostics:Diagnostics.t
   -> symbols_thread:Lev_fiber.Thread.t Lazy_fiber.t
   -> wheel:Lev_fiber.Timer.Wheel.t
+  -> read_dot_merlin:bool
```

Comment:
Could you make this a global variable in `Merlin_config` so that we don't need to pass this everywhere?